src/modules/scraper/adapter/db/deposit-fixture.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Repository } from "typeorm";
+import { Deposit } from "../../model/deposit.entity";
+
+@Injectable()
+export class DepositFixture {
```

Comment:
this is a class used to perform database operations during e2e tests. Each db model will have its own fixture file